spec/main-spec.js

```diff
@@ -0,0 +1,26 @@
+describe("atom-ide-datatip", () => {
+
+    beforeEach(async () => {
+        jasmine.attachToDOM(atom.views.getView(atom.workspace))
+        // Activate activation hook
+        atom.packages.triggerActivationHook('core:loaded-shell-environment');
+        atom.packages.triggerDeferredActivationHooks();
+        // Activate package-deps
+        await atom.packages.activatePackage("busy-signal")
+        await atom.packages.activatePackage("atom-ide-markdown-service")
+    })
+
+    it("Activation", async function () {
+
+        // This makes the log visible again from the command line.
+        spyOn(console, 'log').and.callThrough();
+
+        console.log("\n Activation Started")
```

Comment:
Debug logs? If not, how do we use this info? We know about test case's run and when it's finished (pass/fail).

Reply:
You just run
```
npm run test
```
in your terminal and it shows the result.